docs/migration/css-changes-in-version-4.md

```diff
@@ -5,3 +5,85 @@ title: Changes to Styles in 4.0
 ## Changes in Checkout Components
 
 * `cx-product-variants` selector has been moved to corresponding feature-lib `@spartacus/product`.
+
+## Change in Configurator Attribute Type Components
+
+* `cx-quantity` selector has been added to align its padding to the corresponding attribute type.
+
+## Changes in Configurator Product Title Component
+
+* `width` set to 80% on `%cx-configurator-product-title` to use only 80% of the configuration product title width.
+
+* `button` instead of the anchor link on `%cx-configurator-product-title`.
+
+* `padding` set to 16px/ 32px on `%cx-configurator-product-title` for `cx-details` selector to align spacing depending on the screen size.
+
+* `cx-configurator-image` mixin has been defined on `%cx-configurator-product-title` for `cx-details-image` selector to define the unique styling.
```

Comment:
Please explain what you mean with 'the unique styling'

Reply:
'to achieve a consistent styling'

Reply:
Done!